haas/api.py

```diff
@@ -671,7 +703,9 @@ def switch_register_port(switch, port):
     db.session.commit()
 
 
-@rest_call('DELETE', '/switch/<switch>/port/<path:port>')
+@rest_call('DELETE', '/switch/<switch>/port/<path:port>', Schema({
```

Comment:
Will the framework handle decoding URL argument encodings? I'm just imagining that ports can have all sorts of special characters like spaces, colons and /'s.